docs/website/docs/command-line-interface.md

```diff
@@ -6,4 +6,25 @@ sidebar_position: 7
 
 ## `dlt init`
 
-## `dlt deploy`
\ No newline at end of file
+```
+dlt init <source> <destination>
+```
+
+If you want to start from a [generic template](https://github.com/dlt-hub/python-dlt-init-template),
```

Comment:
Following @TyDunn and @adrianbr suggestions we have two generic templates now
1. **debugging template** with `dlt init <source> <destination>` where the data loading code is commented out, instead you have debug prints and mock data returned so you can run the code right away without credentials setup and see how dlt builiding blocks work
2. **generic template** with ` --generic` with all the data loading code present. you need to implement the missing requests to REST apis and provide credentials to run the pipeline for the first time

@rudolfix change the help text in `dlt` to reflect the name of parameters used here